app_server/app/models/Storage.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Storage class is a singleton object and acts as a facade to the storage
+ * internals.
+ */
+
+var rfr = require('rfr');
+var Utility = rfr('app/util/Utility');
+var logger = Utility.createLogger(__filename);
+
+function Storage() {
+  var Sequelize = require('sequelize');
+  var config = require('./Config');
+  var db = {};
+
+  // initialize database connection
+  var sequelize = new Sequelize(
+    config.name,
+    config.username,
+    config.password, {
+      host: config.host,
+      dialect: config.dialect
+  });
+
+  // load models
+  var models = [
+    'User'
+  ];
+
+  // import each model
+  models.forEach(function(model) {
+    logger.info(model + " imported");
+    db[model] = sequelize.import(__dirname + '/' + model);
+  });
+
+  Object.keys(db).forEach(function(modelName) {
+    if ("associate" in db[modelName]) {
```

Comment:
Er, upon another look, I'm not quite sure what you're doing here.
`db` is an object with its keys connected to each model, right?
So you import models into `db` according to the contents of `models`, linking each of them as keys in `db` connected to a [Model](http://docs.sequelizejs.com/en/latest/api/model/)?
The Model class doesn't have any `associate` property in it, so I'm not quite sure what you're doing in the second forEach loop.